TerrariaServerAPI/TerrariaApi.Server/ServerApi.cs

```diff
@@ -494,5 +500,42 @@ private static bool InvalidateAssembly(Assembly assembly, string fileName)
 
 			return true;
 		}
+
+		private static bool MaliciousAssembly(Assembly assembly)
+		{
+			AssemblyName name = assembly.GetName();
+			Version ver = name.Version;
+
+			var stupidHash = stackoverflowCopypastaFromNazarSHA512(name.Name).ToLower();
```

Comment:
Can we get some better naming on this (both the variable and the method)

Reply:
sure, I'll rename that